src/Microsoft.DotNet.Build.Tasks/PackageFiles/MSBuild.runtimeconfig.json

```diff
@@ -2,7 +2,7 @@
   "runtimeOptions": {
     "framework": {
       "name": "Microsoft.NETCore.App",
-      "version": "1.0.0-rc3-002733"
```

Comment:
We're going to need to update CLI soon in order to support new distros (among other reasons), so it's something we need to do anyways.